### ISSUE TITLE
Latest version of Livechat Widget does not pop in the web page

### ISSUE THREAD

**Issue by PedroPretto** [RocketChat/Rocket.Chat#15203]
### Description:

We have an infrastructure based on multiple rocketchats running in containers installed outside the root(/), after installed update 1.3.0v, the new livechat widget does not pop in the sites. E.g:
domain.com.br/rocketchat1
domain.com.br/rocketchat2

### Steps to reproduce:

1. Set up the rocketchat config on docker
2. Set up the latest version of rocketchat Livechat in your website
3. Go to the web page with embedded livechat widget

### Expected behavior:

The Livechat widget to pop up in the inferior right corner of the web page.

### Actual behavior:

The widget does not show anywhere in the site.

### Server Setup Information:

- Version of Rocket.Chat Server: 1.3.0
- Operating System: Debian 9 (Stretch)
- Deployment Method: Docker
- Number of Running Instances: 3
- MongoDB Version: 4.0

**Comment by renatobecker** [comment]
@PedroPretto Do you get any error on your console log?
Please, check javascript snippet installation, we removed the `1.0.0.` from the public path, so you might have to apply this change on your infrastructure too.

**Comment by ccfiel** [comment]
@renatobecker what do you mean?
from
```
<!-- Start of Rocket.Chat Livechat Script -->
<script type="text/javascript">
(function(w, d, s, u) {
	w.RocketChat = function(c) { w.RocketChat._.push(c) }; w.RocketChat._ = []; w.RocketChat.url = u;
	var h = d.getElementsByTagName(s)[0], j = d.createElement(s);
	j.async = true; j.src = 'https://chat.bai.ph/livechat/1.0.0/rocketchat-livechat.min.js?_=201903270000';
	h.parentNode.insertBefore(j, h);
})(window, document, 'script', 'https://chat.bai.ph/livechat?version=1.0.0');
</script>
<!-- End of Rocket.Chat Livechat Script -->```

to this

```

<script type="text/javascript">
(function(w, d, s, u) {
	w.RocketChat = function(c) { w.RocketChat._.push(c) }; w.RocketChat._ = []; w.RocketChat.url = u;
	var h = d.getElementsByTagName(s)[0], j = d.createElement(s);
	j.async = true; j.src = 'https://chat.bai.ph/livechat/rocketchat-livechat.min.js?_=201903270000';
	h.parentNode.insertBefore(j, h);
})(window, document, 'script', 'https://chat.bai.ph/livechat');
</script>
```

**Comment by repentles** [comment]
chat [](url)

**Comment by Elintondm** [comment]
This post shows the console error using the new html tag with RocketChat installed in a subfolder: https://github.com/RocketChat/Rocket.Chat/issues/14519

Another problem is that running RocketChat in / causes a cross domain error.

**Comment by Elintondm** [comment]
@ccfiel you are right removing 1.0.0 resolves the 404 error, but now we have the cross domain error:

Cross-Origin Read Blocking (CORB) blocked cross-origin response https://www.confirmationcall.com.br:8443/rocketchat/livechat/rocketchat-livechat.min.js?_=201903270000 with MIME type text/html. 

Any tip how to solve this? With the old html tag everything works fine.

**Comment by Elintondm** [comment]
@renatobecker any news about the cross domain error?

**Comment by Elintondm** [comment]
With version 1.3.2 and running Rocketchat in root / with the new widget I get the following strange result:

![image](https://user-images.githubusercontent.com/21336929/63588458-eb8cca80-c57c-11e9-8f52-29a34ff45838.png)

But using the old livechat widget everything works fine:

![image](https://user-images.githubusercontent.com/21336929/63588752-9e5d2880-c57d-11e9-8b4f-ba3b3a83223b.png)


This rocketchat was upgraded from version 0.7. I will test in a news rocketchat installation.

**Comment by Elintondm** [comment]
Running a new installation of rocketchat version 1.3.2 with url configured to (/) NOT a subfolder everything works fine with the new livechat tag.

The problem is upgrading from an old version to 1.3.2. But this is a big problem because I lost the data from the old account.

**Comment by maciej87pl** [comment]
Hello.
Any news about this? I have rocketchat in / folder but livechat widget does not show. I only in console get cross domain error.

**Comment by ramezrafla** [comment]
@ccfiel 
You have a bug in your meter code, you send down the html when we try to load the livechat js

**Comment by ramezrafla** [comment]
Here is how we got around it, we changed our proxy config (nginx) to point directly to the livechat js
Note that we are using snap

```
    location /rocketchat-livechat.min.js {
       default_type "text/javascript";
       root /snap/rocketchat-server/current/programs/web.browser/app/livechat;
    }
```
And in the js included in the page we point directly to `https://<rocketchat-domain>/rocketchat-livechat.min.js`

This is a bug in the code, the original URL returns an HTML page

**Comment by hartois** [comment]
Same problem for me with v1.3.2

**Comment by ramezrafla** [comment]
@renatobecker 
We are all facing the same bug. Take a look at your own site and you will see that `rocketchat-livechat.min.js` return an html page (in fact I don't get how your widget is still up, you are probably loading multiple js scripts)

**Comment by renatobecker** [comment]
> @renatobecker
> We are all facing the same bug. Take a look at your own site (rocket.chat) and you will see that `rocketchat-livechat.min.js` return an html page (in fact I don't get how your widget is still up, you are probably loading multiple js scripts)

What is your current server version?
Can you please share the Javascript snippet you're loading on your website?
